About.html

```diff
@@ -0,0 +1,116 @@
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div id="portfolio" class="wrapper">
+    <header class="nav-container">
+      <div class="top-hader">
+        <nav>
+        <ul class="uls">
+          <li class="social-list glow-on-hover"><a href="#"><img src="images/facebook(1).png" alt="facebook logo"></a></li>
+          <li class="social-list glow-on-hover"><a href="#"><img src="images/twitter.png" alt="twitter logo"></a>
+           </li>
+          <li class="glow-on-hover">English</li>
+          <li class="glow-on-hover">MyPage</li>
+          <li class="glow-on-hover">Layout</li>
+        </ul>
+       </nav>
+       </div>
+       <div class="header2">
+        
+        <nav class="nav">
+           <div class="logoabut">
+            <a href="index.html"> <img class="logo-img1" src="images/logo.png" alt="logo cc Global summit 2015" width="100" height="50"></a>
+             </div>
+             <figure class="humburger">
+              <span  class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+             </figure>
+             <ul class="nav-items" id="nav-items">
+              <li><a href="" class="closebtn">x</a></li> 
+              <li><a href="index.html" class="nav-item ">Home</a> </li>
+              <li><a href="About.html" class="nav-item nav-actives">About</a> </li>
+              <li><a href="#program" class="nav-item">Program</a> </li>
+             <li><a href="#locate" class="nav-item">Sponsor</a> </li>
+             <li><a href="#locate" class="nav-item CcChampaing">CC Champaing</a> </li>
+            </ul>
+        </nav>
+      </div>
+    </header>
+  <main class="cultural-days-wrap" id="main">
+    <section class="about new">
+      <h3 class="about-tite2">"hello! sharing world </h3>
+      <h1 class="about-titel">WORLDWIDE EVENT PROMOTER</h1>
+      <div class="invite-wrap invite-wrap-about">
+        <p class="invite-msg global">International Conference on Climate Change aims to bring together  scientists, researchers and research scholars to exchange and share their experiences and research results.</p>
+      </div>
+      <p class="text-about">Please contact us per Email for any further questions about CC Global Summit 2015!</p>
+      <p class="locate">Conference@newa123</p> 
+    </section>
+    <section class="logo" id="logo" >
+      <h2 class="secondary-tittle logo-tittle">WORLD CONFERENCE ALERTS 2022, 2023 Logo</h2>
+      <div class="bar-icon"></div>
+      <p class="invite-msg logo-msg">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Minima explicabo inventore omnis necessitatibus molestias labore rerum beatae consequuntur, et eius vitae recusandae, quis quidem. Minus fugit excepturi aut nemo mollitia.</p>
+      <img class="logo-img" src="images/logo.png" alt="logo " width="572" height="206">
+    </section>
+    <a href="#locate"></a><div  class="separeter"></div>
+    
+    <section class="logo" id="gallery">
```

Comment:
- [ ] Again for this section, the contents are not properly aligned to the center as is in the given design, kindly fix this
- [ ] Replace the placeholder text with actual content as this is a requirement
- [ ] follow the margins and padding of the given design as this is a requirement as well

Your design
![image](https://user-images.githubusercontent.com/81932478/204781357-8f6b1b0a-edf9-41d4-b1a6-d8236d3c8e73.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/204781493-b2fdf07e-9d1f-4310-bde1-97587ee71006.png)